boot/makebootable.go

```diff
@@ -48,19 +48,20 @@ type BootableSet struct {
 	Recovery bool
 }
 
-// MakeBootable sets up the given bootable set and target filesystem
+// MakeBootableImage sets up the given bootable set and target filesystem
 // such that the system can be booted.
 //
-// rootdir points to an image filesystem (UC 16/18), image recovery
-// filesystem (UC20 at prepare-image time) or ephemeral system (UC20
-// install mode).
-func MakeBootable(model *asserts.Model, rootdir string, bootWith *BootableSet, sealer *TrustedAssetsInstallObserver) error {
+// rootdir points to an image filesystem (UC 16/18) or an image recovery
+// filesystem (UC20 at prepare-image time).
+// It should not point to an ephemeral system during UC20 install mode as
```

Comment:
this part of the comment is sligthly confusing,  naively one would expect to mention the actual run root system but is not what is used anyway as per my other comment, it should probably refer to the fact that on UC20 bootWithRecovery is expected to be true maybe?

Reply:
sure I tried to rephrase this hopefully it makes sense